examples/asset/utilization/README.md

```diff
@@ -0,0 +1,34 @@
+Asset Utilization Example
+=====================
+
+This is a basic example console application demonstrating how to track asset utilization
+using SystemLink Asset Utilization APIs.
+
+Running the Example
+-------------------
+
+1. Download and extract the [repository source](https://github.com/ni/systemlink-client-docs/archive/master.zip).
+2. Install the [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core).
+3. Navigate to the example's directory and use the [`dotnet run` command](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-run?tabs=netcore21).
+
+To run the example, SystemLink Client needs to be installed. 
```

Comment:
More specifically, it needs to be a managed system to actually get results, right? Maybe you could re-use wording from the configuration example: "Run the example on a machine with SystemLink Client and SystemLink Server installed, or on a system managed by a SystemLink Server."

Reply:
One could store utilization data without being a managed system. But I will add this information to make it more clear what the behavior is.